ksql-engine/src/main/java/io/confluent/ksql/planner/plan/AggregateNode.java

```diff
@@ -369,4 +370,76 @@ private Schema buildAggregateSchema(
     return schemaBuilder.build();
   }
 
+  private List<Expression> getInternalExpressionList(
```

Comment:
This should really be a method of `InternalSchema`. Maybe call it `InternalSchema.getInternalExpressions(List<Expression> expressions)`.